components/security-mgt/org.wso2.carbon.security.mgt/src/main/java/org/wso2/carbon/security/SecurityConstants.java

```diff
@@ -119,4 +119,76 @@
 
     private SecurityConstants(){}
 
+    /**
+     * Contains the Keystore management service related constants.
+     */
+    public static class KeyStoreMgtConstants {
+
+        public static final String FILTER_FIELD_ALIAS = "alias";
+        public static final String FILTER_OPERATION_EQUALS = "eq";
+        public static final String FILTER_OPERATION_STARTS_WITH = "sw";
+        public static final String FILTER_OPERATION_ENDS_WITH = "ew";
+        public static final String FILTER_OPERATION_CONTAINS = "co";
+
+        public static final String SERVER_TRUSTSTORE_FILE = "Security.TrustStore.Location";
+
+        /**
+         * Enum for Keystore management service related errors.
+         */
+        public enum ErrorMessage {
+
+            /**
+             * Server errors.
+             */
+            ERROR_CODE_RETRIEVE_KEYSTORE("KSS-65001",
+                    "Unable to retrieve the keystore for tenant: %s."),
+            ERROR_CODE_RETRIEVE_KEYSTORE_INFORMATION("KSS-65002",
+                    "Unable to retrieve keystore information for keystore: %s"),
+            ERROR_CODE_RETRIEVE_CLIENT_TRUSTSTORE("KSS-65003",
+                    "Unable to retrieve client truststore for tenant: %s"),
+            ERROR_CODE_RETRIEVE_CLIENT_TRUSTSTORE_ALIASES("KSS-65004",
+                    "Unable to retrieve the client truststore aliases for tenant: %s."),
+            ERROR_CODE_RETRIEVE_CLIENT_TRUSTSTORE_CERTIFICATE("KSS-65005",
+                    "Unable to retrive the client truststore certificate for alias: %s."),
+            ERROR_CODE_ADD_CERTIFICATE("KSS-65006",
+                    "Unable to add certificate with alias: %s"),
+            ERROR_CODE_DELETE_CERTIFICATE("KSS-65007",
+                    "Unable to delete certificate with alias: %s"),
+            ERROR_CODE_VALIDATE_CERTIFICATE("KSS-65008", "Error occurred while validating the " +
+                    "certificate."),
+            /**
+             * Client error.
+             */
+            ERROR_CODE_CERTIFICATE_EXISTS("KSS-60001",
+                    "Certificate already exists with alias: %s"),
```

Comment:
A better message would be "Provided certificate already exists with the alias: %s"

Reply:
fixed